src/featurelist.cpp

```diff
@@ -153,8 +153,7 @@ bool FeatureList::accountCreationInAppSupported() const {
 }
 
 bool FeatureList::inAppPurchaseSupported() const {
-#if defined(MVPN_IOS)
-  // Soon, android will have IAP.
+#if defined(MVPN_IOS) || defined(MVPN_ANDROID)
```

Comment:
This needs to be rebased. Apply the same change to `src/features/featureinapppurchase.h`